src/figvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/figvalidationservice/ValidateApplicantForSharingController.java

```diff
@@ -0,0 +1,44 @@
+package ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice.exception.FigaroValidationServiceException;
+import ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice.types.ValidateApplicantForSharingRequest;
+import ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice.types.ValidateApplicantForSharingResponse;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+
+/**
+ * 
+ * ValidateApplicantForSharingController class to handle requests for /validateApplicantForSharing
+ * 
+ * @author archana
+ *
+ */
+@RestController
+public class ValidateApplicantForSharingController {
+
+	@Autowired
+	private FigaroValidationImpl figservice; // connection to ORDS client.
+
+	@RequestMapping(value = "/validateApplicantForSharing", produces = {
+			"application/xml" }, method = RequestMethod.GET)
+	@ApiOperation(value = "Validate Applicant For Sharing", notes = "", response = ValidateApplicantForSharingResponse.class, tags={ "Figaro Validation Services"})
+  	@ApiResponses(value = {@ApiResponse(code = 200, message = "Successful operation", response = ValidateApplicantForSharingResponse.class) })
+    
+	public ValidateApplicantForSharingResponse validateApplicantForSharingResponse(
+			@ApiParam(value = "applPartyId", required = false) @RequestParam(value = "applPartyId", defaultValue = "0",required = false) String applPartyId,
+			@ApiParam(value = "jurisdictionType", required = false) @RequestParam(value = "jurisdictionType", defaultValue = "",required = false) String jurisdictionType)
+			throws FigaroValidationServiceException {
+
+		return figservice
+				.validateApplicantForSharing(new ValidateApplicantForSharingRequest(applPartyId, jurisdictionType));
```

Comment:
This area will need try/catch and logging when you get to it.

Reply:
logging,test-cases and exception handling will be included in next PR.

Reply:
in future work you should outline working to come. I should add a section for this to the pull request template. it's a bit of a side effect of doing small PRs consistently I think.

Reply:
Great idea. This will be an ongoing situation.

Reply:
good idea.I was thinking of adding in the description section .will do in future PR's

Reply:
logging and test case included.